backends/github/gist/github-gist.js

```diff
@@ -103,18 +106,12 @@ export default class GithubGist extends Github {
 	 * Parse Gist URLs, return username, gist id, filename
 	 */
 	static parseURL (source) {
-		let ret = super.parseURL(source);
-		let path = ret.url.pathname.slice(1).split("/");
-
-		ret.owner = path.shift();
-		ret.gistId = path.shift();
+		let ret = Object.assign({}, super.parseURL(source));
```

Comment:
We don't need to clone here, `super.parseURL()` should already return a new object. In other places I used this pattern because they included specific properties.

Reply:
RIght. Sometimes, my brain simply refuses to work correctly late in the evening. lol